src/shared/components/ProfilePage/Stats/StatsModal/index.jsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-env browser */
+import React from 'react';
+import PT from 'prop-types';
+import X from 'assets/images/profile/x-mark-gray.svg';
+import { getRatingColor } from 'utils/tc';
+import StatsCategory from '../../StatsCategory';
+import styles from './styles.scss';
+
+
+class StatsModal extends React.Component {
+  componentDidMount() {
+    document.body.classList.add(styles['is-modal-open']);
```

Comment:
This is not ReactJS way to do it.

Reply:
Since I cannot reuse `Modal` component in this modal, I reuse its body style now.

https://github.com/topcoder-platform/community-app/blob/develop/src/shared/components/Modal/index.jsx